ibctest/relayer_local.go

```diff
@@ -0,0 +1,555 @@
+package ibctest
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/ibc-go/v5/modules/core/03-connection/types"
+	"github.com/cosmos/relayer/v2/cmd"
+	"github.com/cosmos/relayer/v2/relayer"
+	"github.com/cosmos/relayer/v2/relayer/chains/cosmos"
+	"github.com/strangelove-ventures/ibctest/v5/ibc"
+	"go.uber.org/zap"
+)
+
+type LocalRelayer struct {
```

Comment:
I see this is different from ibctest/relayer.go but I don't immediately see why a second version is necessary. The other version goes through the cobra objects, which more closely resembles production usage.

Reply:
Good call, I thought we would miss the race detection with going through the `RunC` methods but I see now that they will be detected. Updated